app/services/action_creator.rb

```diff
@@ -20,6 +20,11 @@ def initialize(params)
   def run
     params[:params][:mailing_id] = extract_mailing_id(params[:params][:akid])
 
+    if params[:params][:country] == 'United Kingdom'
```

Comment:
If we're going to do this for more than just the UK in the future, having an `if / else if / else if ...` block here can become messy. It would be better if we had a separate module that handled this... e.g.

```ruby

def run
  CountryService.extend_with_local_data(params)
  ...
end
```

And we could then have a hash / map that contains what we call for each country: 

```rb
{ country: 'United Kingdom', fn: lambda { |postal| GetConstituency.for(postal) } }
```